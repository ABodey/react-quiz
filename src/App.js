import React, { Component } from 'react';
import './App.css';
import Books from './Books';

class App extends Component {
  render() {
    return (
      <div className="App">
      This is the App for your books:
        <Books />
      </div>
    );
  }
}

export default App;
