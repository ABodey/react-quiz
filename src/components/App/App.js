import React, { Fragment } from 'react'
import Books from 'components/Books/Books'


const App = () => (
  <Fragment>
  <h1> hello world </h1>
  <Books/>
  </Fragment>
)

export default App
