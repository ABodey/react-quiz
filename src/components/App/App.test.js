import React from 'react'
import { mount } from 'enzyme'
import App from 'components/App/App'

describe('<App />', () => {
    it ('should match snapshots', () => {
      const wrapper = mount (<App />)

      expect(wrapper).toMatchSnapshot()
    })
})

