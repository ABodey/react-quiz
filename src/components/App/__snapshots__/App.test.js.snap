// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snapshots 1`] = `
<App>
  <h1>
     React Quiz 
  </h1>
  <Books>
    <p>
      You dont have any books
    </p>
    <AddBook
      onAddBook={[Function]}
    >
      <form
        onSubmit={[Function]}
      >
        <input
          name="title"
          placeholder="Title"
          required={true}
          type="text"
        />
        <input
          name="author"
          placeholder="Author"
          required={true}
          type="text"
        />
        <button
          type="submit"
        >
          Add Book
        </button>
      </form>
    </AddBook>
  </Books>
</App>
`;
